lua/dashboard/theme/hyper.lua

```diff
@@ -349,11 +355,6 @@ local function theme_instance(config)
     gen_footer(config)
     map_key(config, '<CR>')
     require('dashboard.events').register_lsp_root(config.path)
-    local size = math.floor(vim.o.lines / 2)
```

Comment:
why remove the auto pad at head

Reply:
I am busy adding the option to manually pad the top.  And the existing code did not work well with the padding.

Reply:
> Default setup: ![image](https://user-images.githubusercontent.com/81622310/215734365-7a6a9f17-79c3-4787-8517-8dc61ab60415.png)
> 
> Add 5 padding to each segment: ![image](https://user-images.githubusercontent.com/81622310/215734666-27fa4696-f415-464d-b7a0-bd365f8be4e1.png)
> 
> I now realize the centering does not take the padding into account.

See here.

Reply:
Will add centering that takes padding into account.